### DISCUSSION TITLE
Decrease number of issues for "next"

### DISCUSSION THREAD

**Discussion by cronventis** [rancher-sandbox/rancher-desktop#2074] [Ideas]
So I'm currently waiting for 1.2.2, to fix the bug on windows with the ssl cert buffer.
I was hoping this release would come out quickly, since I was not able to install rancher desktop at all.
Today, I went on and tried to find out when the next release would come, and I found a huge list of open tickets for the "[next](https://github.com/rancher-sandbox/rancher-desktop/milestone/13)" milestone.

Would it be possible to increase the frequency of releases, so that smaller fixes come more frequent?

**Comment by jandubois** [comment]
> Today, I went on and tried to find out when the next release would come, and I found a huge list of open tickets for the "[next](https://github.com/rancher-sandbox/rancher-desktop/milestone/13)" milestone.

The milestones are just an internal tool to narrow the scope of things we are looking at during a release cycle. It does not represent a promise of what will actually make it in, and we typically move more and more things from "Next" to "Later" as time progresses.

> Would it be possible to increase the frequency of releases, so that smaller fixes come more frequent?

I don't think so: we had 7 releases in the 3½ months in this year, including one beta and 3 patch releases.

Realistically I would not expect to be able to do more than 8 to 10 minor releases of the 1.X.0 variant per year, as there is a certain amount of overhead and extended manual testing involved each time.

If you don't care about that extra round of QA, then you can always download builds from CI; there are Github Action binaries from every merge to the `main` branch. They will not be signed though, as Github has no access to our signing keys.

Anyways, there will very likely be a new release next week; so hang in there! 😄

**Reply by cronventis** [reply]
Alright.. Thanks for the explanation :)
